helper-cli/src/main/kotlin/commands/packageconfig/CreateCommand.kt

```diff
@@ -100,8 +100,17 @@ internal class CreateCommand : CliktCommand(
     private val nonOffendingLicenseCategories by option(
         "--non-offending-license-categories",
         help = "Configure the path exclude generation to not create excludes for files or directories which only " +
-                "contain non-offending licenses specified by this comma separated list of license category names. " +
-                "Each category name must be present in the given license classifications file."
+                "contain non-offending licenses specified by this comma separated list of license category names and " +
+                "by the option '--non-offending-license-ids'. Each category name must be present in the given " +
+                "license classifications file."
+    ).split(",").default(emptyList())
+
+    private val nonOffendingLicenseIds by option(
+        "--non-offending-license-ids",
+        help = "Configure the path exclude generation to not create excludes for files or directories which only " +
+                "contain non-offending licenses specified by this comma separated list of license IDs and by " +
+                "the option '--non-offending-license-categories'. Each category name must be present in the given " +
+                "license classifications file."
```

Comment:
The last sentence does not fit here, and maybe just say "cumulative with '--non-offending-license-categories'".